pkg/controllers/webhook/controller.go

```diff
@@ -651,7 +656,10 @@ func (c *controller) buildResourceMutatingWebhookConfiguration(ctx context.Conte
 					}
 				}
 			}
+			rules := p.GetSpec().Rules
+			operationStatusMap = computeOperationsForMutatingWebhookConf(rules, operationStatusMap)
```

Comment:
Don't you need to filter out rule types other than mutate and verifyImages?

Reply:
```
		for _, p := range policies {
			if p.AdmissionProcessingEnabled() {
				spec := p.GetSpec()
				if spec.HasMutate() || spec.HasVerifyImages() {
					if spec.GetFailurePolicy(ctx) == kyvernov1.Ignore {
						c.mergeWebhook(ignore, p, false)
					} else {
						c.mergeWebhook(fail, p, false)
					}
				}
			}
			rules := p.GetSpec().Rules
			operationStatusMap = computeOperationsForMutatingWebhookConf(rules, operationStatusMap)
	}
``` 
We are going through all policies and checking for Operations. I think earlier I had put it with the if block 
```
				if spec.HasMutate() || spec.HasVerifyImages() {
					if spec.GetFailurePolicy(ctx) == kyvernov1.Ignore {
						c.mergeWebhook(ignore, p, false)
					} else {
						c.mergeWebhook(fail, p, false)
					}
				}
```
Now this looks okay, right?

Reply:
`buildResourceMutatingWebhookConfiguration ` builds the resource mutating webhook configuration for relevant rules, while you select all types of rules [here](https://github.com/kyverno/kyverno/pull/8437/files#diff-6c0dd4f85a678b8261eb41e6ab885470b2d6633cb9d5728bb31dae8e532ef85fR659). It will cause invalid configurations for the operations field.

Reply:
Understood, thanks @realshuting 
Pushing this change